docker-compose-deploy.yml

```diff
@@ -0,0 +1,108 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres
+    environment:
+      - POSTGRES_DB=${SQL_DATABASE}
+      - POSTGRES_USER=${SQL_USER}
+      - POSTGRES_PASSWORD=${SQL_PASSWORD}
+      - PGHOST=${SQL_HOST}
+      - PGPORT=${SQL_PORT}
+      - PGUSER=${SQL_USER}
+      - PGDATABASE=${SQL_DATABASE}
+      - PGPASSWORD=${SQL_PASSWORD}
+    volumes:
+      - /home/github-runner/podman-volumes/db-data:/var/lib/postgresql/data
+      - /home/github-runner/podman-volumes/db-backups:/backups
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      start_period: 5s
+      interval: 5s
+      timeout: 5s
+      retries: 5
+  ip2country:
+    image: extrawurst/ip2country:latest
+    ports:
+      - 8854:5000
+  server:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    depends_on:
+      db:
+        condition: service_healthy
+      ip2country:
+        condition: service_started
+    volumes:
+      - /home/github-runner/podman-volumes/server-static:/server/static
+      - /home/github-runner/podman-volumes/server-logs:/server/logs
+      - /home/github-runner/podman-volumes/server-uploads:/server/upload
+    environment:
+      - AML_ALLOWED_HOSTS=${AML_ALLOWED_HOSTS}
+      - AML_DEBUG=${AML_DEBUG}
+      - AML_CORS_ORIGIN_WHITELIST=${AML_CORS_ORIGIN_WHITELIST}
+      - AML_LOCATION_PROVIDER=${AML_LOCATION_PROVIDER}
+      - AML_SECRET_KEY=${AML_SECRET_KEY}
+      - AML_SUBPATH=${AML_SUBPATH}
+      - DJANGO_SETTINGS_MODULE=${DJANGO_SETTINGS_MODULE}
+      - DJANGO_SUPERUSER_USERNAME=${DJANGO_SUPERUSER_USERNAME}
+      - DJANGO_SUPERUSER_EMAIL=${DJANGO_SUPERUSER_EMAIL}
+      - DJANGO_SUPERUSER_PASSWORD=${DJANGO_SUPERUSER_PASSWORD}
+      - SENTRY_DSN=${SENTRY_DSN}
+      - SQL_DATABASE=${SQL_DATABASE}
+      - SQL_USER=${SQL_USER}
+      - SQL_PASSWORD=${SQL_PASSWORD}
+      - SQL_HOST=${SQL_HOST}
+    ports:
+      - 8000:8000
+    command: bash -c "python manage.py migrate && python manage.py bootstrap && python manage.py collectstatic --noinput && gunicorn aml.wsgi:application --bind 0.0.0.0:8000"
```

Comment:
Just an idea, we can also do this later: we might start up gunicorn with multiple workers, which may help us with high server loads.